GitUI/UserControls/RevisionGrid/Graph/RevisionGraphRow.cs

```diff
@@ -185,6 +185,7 @@ public int GetLaneIndexForSegment(RevisionGraphSegment revisionGraphRevision)
 
         public void MoveLanesRight(int fromLane)
         {
+            _gaps ??= new();
             int nextGap = _gaps.Min(lane => lane > fromLane ? lane : null) ?? int.MaxValue;
```

Comment:
```suggestion
            int nextGap = _gaps?.Min(lane => lane > fromLane ? lane : null) ?? int.MaxValue;
```

And move:

```c#
            _gaps ??= new();
```

Down before the first actual write on line 205.

Reply:
Done because it is clearer. Since this function is called only if there are lanes to be moved, it has no effect on the number of allocations.